sgid/state-only/index.md

```diff
@@ -10,6 +10,10 @@ title: How to Connect to the SGID via ArcSDE
 layout: page_nocrumb
 categories: []
 ---
+
+**Heads up!** We will be replacing the state-only ArcSDE-based SGID10 database with the [Open SGID]({% link sgid/open-sgid/index.md %}) on **December 31st, 2020**. After this date, the SGID10 database will no longer be accessible. The PostGIS-based Open SGID contains the same datasets and is available now to anyone with an internet connection. Please take time now to update your data references.
```

Comment:
```suggestion
**Heads up!** We will be replacing the state-only ArcSDE-based SGID10 database with the [Open SGID]({% link sgid/open-sgid/index.md %}) on **December 31st, 2020**. After this date, the SGID10 database will no longer be accessible. The Open SGID contains the same datasets and is available now to anyone with an internet connection. Please take time now to update your data references.
```

Reply:
I think the differences between SDE and PostGIS are important enough to mention, but linking to the blogpost may be more effective than including them here. Let me know what you think about the next commit.

Reply:
Also, it might make sense to flesh out sgid/open-sgid with some of the info from the blog post (primarily the differences between PostGIS and SDE and more explanation about the github repo). If anyone sees the Cheshire Cat, let me know.